slasher/main.go

```diff
@@ -73,6 +75,13 @@ func main() {
 	app.Flags = appFlags
 	app.Action = startSlasher
 	app.Before = func(ctx *cli.Context) error {
+		// Load any flags from file, if specified.
+		if ctx.IsSet(cmd.ConfigFileFlag.Name) {
+			if err := altsrc.InitInputSourceWithContext(appFlags, altsrc.NewYamlSourceFromFlagFunc(cmd.ConfigFileFlag.Name))(ctx); err != nil {
```

Comment:
Line is too long, probably sth like:

```suggestion
			createSourceFunc := altsrc.NewYamlSourceFromFlagFunc(cmd.ConfigFileFlag.Name)
			if err := altsrc.InitInputSourceWithContext(appFlags, createSourceFunc)(ctx); err != nil {			
```

Reply:
thanks